app/shop.py

```diff
@@ -0,0 +1,36 @@
+import math
+
+from datetime import datetime
+from dataclasses import dataclass
+from app.customer import Customer
+
+
+@dataclass
+class Shops:

```

Comment:
```suggestion
class Shop:
```
class names must be singular